Choreographer_30FPS/app/src/main/jni/Common/ndk_helper/gl3stub.h

```diff
@@ -0,0 +1,659 @@
+#ifndef __gl3_h_
+#define __gl3_h_
```

Comment:
Not for now as gl3stub.h is defining functions differently than the gl3.h.
Will revisit the issue later.